issue-template.md

```diff
@@ -0,0 +1,28 @@
+The following is a template for issues raised in Github repos.
+
+The first three questions should always be answered, the remaining are subject to requirement.
+Be as specific as possible and provide links to other discussions/issues/PRs etc where possible.
+
+If you are creating an tech debt with your work then create issues to be resolved for this and list them in the appropriate section.
+
+If you know the work will be undertaken by a specific POD then add a label with the POD name to the issue.
+
+If you are creating multiple issues around a topic or with a common goal then create a milestone and add all relevant issues to it.
+
+Please, when you begin working on an issue assign yourself to it and treat the issue as a mind-dump. Write all problems/ideas/blockers/designs/failures etc. that you have on there so that people can easily follow decisions you have made and discussion you have had. Try to update your issues at least once a day.
+
+```
+#What is the problem you're trying to solve?
+
+#What is the benefit in solving it?
```

Comment:
I would suggest using `customer benefit` here as we have a high focus on customer impact of our work?